nlu.py

```diff
@@ -20,6 +20,38 @@ def _get_commands(text_sequence: [str]) -> set:
     return possible_commands
 
 
+def _consine_similarity(a: [str], b: [str]) -> float:
+    words = set(a+b)
+    vector_a = _word2vector(words, a)
+    vector_b = _word2vector(words, b)
+    return _cosine_theta(vector_a, vector_b)
+
+
+def _cosine_theta(a: [int], b: [int]):
+    numerator = sum(x1 * x2 for x1, x2 in zip(a, b))
+    denominator = _dot(a)*_dot(b)
```

Comment:
`expression1 * expression2`

Reply:
Does that format by formatter?

Reply:
Which format?

Reply:
For me, the operator should have space for expression like what I show

Reply:
oh, I am not using any formatter in my IDE...
wait me for a second.

Reply:
Do you use PyCharm? You can add autopep8  for on save hook to get a auto formatter

Reply:
And I guess add git hook pre-commit would be a good idea

Reply:
yes, I am installing now.

Reply:
done.